ksqldb-engine/src/main/java/io/confluent/ksql/engine/rewrite/AstSanitizer.java

```diff
@@ -123,8 +124,11 @@ protected Optional<AstNode> visitInsertInto(
             .map(
                 l -> new NodeLocation(
                     l.getLineNumber(),
-                    l.getColumnNumber() + "INSERT INTO".length()
-                )
+                    l.getColumnNumber() + "INSERT INTO".length(),
+                    OptionalInt.empty(),
+                    OptionalInt.empty(),
+                    OptionalInt.empty(),
+                    OptionalInt.empty())
```

Comment:
Meh, I wouldn't change the constructors that can just use the "old" constructor.